Automattic/pocket-casts-android
# Migrate profile sections to Compose UI

## Description

This migrates the Profile tab sections to Compose UI.

## Testing Instructions

1. Go to the Profile tab.
2. Verify that the you can navigate to different sections of the app by tapping on the sections.
3. Verify that tapping on section items trigger section navigation events.
4. Check the sections UI with different themes.
5. Check the sections UI in the landscape orientation.

## Screenshots or Screencast 

![Screenshot 2024-11-19 at 09 39 25](https://github.com/user-attachments/assets/70342765-62a9-4ea0-8a5e-aa57993da773)

## Checklist
 
#### I have tested any UI changes...
